Other/WebApplication/WebApplication/Global.asax.cs

```diff
@@ -38,23 +38,22 @@ void Application_Start(object sender, EventArgs e)
             Environment.SetEnvironmentVariable(@"PATH", path, EnvironmentVariableTarget.Process);
             
             //
-            // Initialize DLE...
+            // Initialize DLE
```

Comment:
If we're expunging all references to "DLE", this could just read "Initialize Library"

Reply:
removed